### ISSUE TITLE
Support overlays

### ISSUE THREAD

**Issue by tpikonen** [rinigus/pure-maps#75]
Would it be possible to support (raster) overlays like 'Poor Maps' does?

**Comment by rinigus** [comment]
Yes, I was thinking about it too. I am planning to group maps and then overlays would be a possible extension. Its not going to be the first issue I'll address, but lets plan it

**Comment by rinigus** [comment]
Which overlays do you have in mind?

**Comment by tpikonen** [comment]
At least OpenPTmap would be useful.

**Comment by rinigus** [comment]
In case of OpenPTMap, I think we should actually implement our own dataset import. Not sure where do they get timetable, but looks like all other public transport lines come from OSM database. So, it would require loading corresponding data into vector map tiles and then making style for public transport rendering. With the new map style groups, it would naturally will fit into the application. 

Any other overlays?
